### ISSUE TITLE
Hang up when when mounting second export.

### ISSUE THREAD

**Issue by Yasushi** [Yasushi/winnfsd#5]
## Reproduce

pathfile.txt

```
d:\export\foobar
d:\export\baz
```

start winnfsd.

```
winnfsd -pathFile d:\export\pathfile.txt -id 0 0
```

mount

```
sudo mount -t nfs -o vers=3,rw,udp irpa:/D/export/foobar /mnt/1
sudo mount -t nfs -o vers=3,rw,udp irpa:/D/export/baz /mnt/2
```
## Causes

Crashed at https://github.com/Yasushi/winnfsd/blob/bb328cba4ed1559ce13bba8dfabf38c8b65c98af/src/FileTree.cpp#L173 .

FileTree has only one root. Therefore, it will fail in the second.

**Comment by Yasushi** [comment]
CandoImage/winnfsd@939e8779c28f71b4b8647cb97b3438c012efde7b

**Comment by Yasushi** [comment]
merged 43092f2352b2427a0445ee77dd35b5afb83d0519
